apstools/devices/lakeshore_controllers.py

```diff
@@ -215,68 +160,9 @@ def __init__(self, *args, loop_number=None, **kwargs):
         self.loop_number = loop_number
         super().__init__(*args, **kwargs)
 
-
-class LakeShore336_LoopBase(PVPositionerSoftDoneWithStop):
-    """
-    One control loop of the LakeShore 336 temperature controller.
-
-    Each control loop is a separate process controller.
-
-    DEFAULTS:
-
-    - timeout: 10 hours
-    - tolerance: 0.1
-    """
-
-    readback = FormattedComponent(
-        EpicsSignalRO, "{prefix}IN{loop_number}", auto_monitor=True, kind="hinted"
-    )
-    setpoint = FormattedComponent(
-        EpicsSignalWithRBV,
-        "{prefix}OUT{loop_number}:SP",
-        put_complete=True,
-        kind="normal",
-    )
-
-    def __init__(self, *args, loop_number=None, timeout=10 * HOUR, **kwargs):
-        self.loop_number = loop_number
-        super().__init__(*args, timeout=timeout, tolerance=0.1, **kwargs)
-
-
-class LakeShore336_LoopMore(LakeShore336_LoopBase):
-    """
-    Additional controls for LakeShore 336 loop1 and loop2: heater, pid, & ramp.
-
-    Only used on loops 1 & 2.
-    """
-
-    heater = FormattedComponent(
-        EpicsSignalRO, "{prefix}HTR{loop_number}", auto_monitor=True, kind="normal"
-    )
-    heater_range = FormattedComponent(
-        EpicsSignalWithRBV,
-        "{prefix}HTR{loop_number}:Range",
-        kind="config",
-        auto_monitor=True,
-        string=True,
-    )
-
-    pid_P = FormattedComponent(
-        EpicsSignalWithRBV, "{prefix}P{loop_number}", kind="config"
-    )
-    pid_I = FormattedComponent(
-        EpicsSignalWithRBV, "{prefix}I{loop_number}", kind="config"
-    )
-    pid_D = FormattedComponent(
-        EpicsSignalWithRBV, "{prefix}D{loop_number}", kind="config"
-    )
-
-    ramp_rate = FormattedComponent(
-        EpicsSignalWithRBV, "{prefix}RampR{loop_number}", kind="config"
-    )
-    ramp_on = FormattedComponent(
-        EpicsSignalWithRBV, "{prefix}OnRamp{loop_number}", kind="config"
-    )
+    @property
+    def egu(self):
```

Comment:
This is syntactic sugar but inconsistent in apstools.  Is it necessary to meet a particular interface?  Otherwise, it becomes a special case that needs additional explanation (with _this_ support, use `.egu` but with other devices, use `.units.get()`).

I'm requesting this property be removed unless absolutely necessary.